app-modules/report/tests/Tenant/Filament/Widgets/StudentInteractionUsersTableTest.php

```diff
@@ -371,3 +374,34 @@
         ]))
         ->assertCanNotSeeTableRecords(collect([$userWithoutInteractions]));
 });
+
+it('has an export action', function () {
+    livewire(StudentInteractionUsersTable::class, [
+        'cacheTag' => 'report-student-interaction',
+        'filters' => [],
+    ])->assertTableActionExists(ExportAction::class);
+});
+
+it('can start an export, sending a notification', function () {
+    Storage::fake('s3');
+
+    $user1 = User::factory()->create();
+
+    $student = Student::factory()->create();
+
+    Interaction::factory()
+        ->count(5)
+        ->for($student, 'interactable')
+        ->for($user1, 'user')
+        ->create();
+
+    asSuperAdmin();
+
+    livewire(StudentInteractionUsersTable::class, [
+        'cacheTag' => 'report-student-interaction',
+        'filters' => [],
+    ])
+        ->callTableAction(ExportAction::class)
```

Comment:
It looks like in Filament v4 this is no deprecated and should be replaced with `callAction`

Reply:
@danharrin could this be an issue with the Filament methods we are told to switch to? Or perhaps are we now registering the actions wrong?